main.go

```diff
@@ -79,6 +83,21 @@ func (m *HarborCli) BuildHarbor(ctx context.Context, directoryArg *dagger.Direct
 	return outputs
 }
 
+// Return a container that build the Harbor binary for development purposes.
+func (m *HarborCli) BuildDev(ctx context.Context, directoryArg *dagger.Directory) *dagger.Container {
+    fmt.Println("🛠️  Building Harbor binary with Dagger...")
+
+    // Define the path for the binary output
+    binaryOutputPath := "/src/bin/harbor"
+    return dag.Container().
+        From("golang:latest").
+        WithMountedDirectory("/src", directoryArg).
+        WithWorkdir("/src/cmd/harbor").
+        WithExec([]string{"go", "build", "-o", binaryOutputPath, "main.go"}).
+        WithWorkdir("/src")
+}
```

Comment:
- Update build-dev to produce binary out of the box without any args or flags such as directory.
- `dagger call build-dev` should work.

![Screenshot_2024-08-16_18-37-51](https://github.com/user-attachments/assets/00832e79-f772-4393-8970-405734c2d103)